### PR TITLE
Add scripts to gather airdrop addresses

### PR DESCRIPTION
**Pull request by smb2796**
This PR adds initial versions of all necessary scripts to gather airdrop addresses.

**Summary**

- `collectAddresses.js`: master script to run all of the other scripts from.
- `GetGovernanceParticipants.js`: Combines and de-dupes results from various snapshot organizations.
- `GetAllTokenholders.js`: Submits a BQ Query, `TokenBalanceQuery.js`, to gather a list of addresses that held 10 or more UMA at `2021-02-10 05:00:49 UTC` (block 11830000).
- `GetAllSponsors.js`: Mostly a duplication of the existing `GetAllSponsors.js` script [here](https://github.com/UMAprotocol/protocol/blob/master/packages/core/scripts/mainnet/GetAllSponsors.js). Returns all sponsors who created a position before block 11830000.
- `GetAllVoters.js`: Based off of `GetAllSponsors.js`. Returns all voters who voted before block 11830000.

**Running the scripts**

To run `collectAddresses.js` you will need to set a `GOOGLE_AUTHENTICATION_CREDENTIALS` env variable. After this is done, run:
```
node collectAddresses.js --url "INFURA_URL"
```

**Outstanding Items**
- ~Fix `GetAllVoters.js`.~
- ~Combine results from `GetAllVoters.js` and `GetAllSponsors.js`.~
- ~Run all scripts in one workflow~

### CODE REVIEW

**Review by mrice32 — COMMENTED** [review]
Looks great! Mostly questions around the maker governance queries.

**Review by daywiss — APPROVED** [review]
nice work with this, looks good to me

**Review by mrice32 — APPROVED** [review]
LGTM!

**Review by pemulis — APPROVED** [review]

File: collectAddresses.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    console.log("Running airdrop queries");
+
+    let SPONSORS_AND_VOTERS = {};
+
+    await getGovernanceParticipants();
+    await getAllTokenholders();
+    // const voters = await getAllVoters();
+    const sponsors = await getAllSponsors(argv.url);
+
+    sponsors.forEach((sponsor) => {
```
[the remark this answers is not in the document]
Comment (rk391339) [reply collectAddresses.js:34]: Similer

**Pull request merged**